model/task/db.go

```diff
@@ -477,7 +477,10 @@ func scheduleableTasksQuery() bson.M {
 		PriorityKey: bson.M{"$gte": 0},
 
 		//Filter tasks containing unattainable dependencies
-		bsonutil.GetDottedKeyName(DependsOnKey, DependencyUnattainableKey): bson.M{"$ne": true},
+		"$or": []bson.M{
+			{bsonutil.GetDottedKeyName(DependsOnKey, DependencyUnattainableKey): bson.M{"$ne": true}},
+			{OverrideDependenciesKey: true},
```

Comment:
is this indexed?

Reply:
The last two predicates are not indexed, but the first two (activated and status) are.
We had decided to leave it (https://github.com/evergreen-ci/evergreen/pull/2602#discussion_r318042330) given that the index is relatively selective.